src/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './GoodsList.scss';
+
+type Props = {
+  goods: Good[],
+  errors: boolean,
+};
+
+export const GoodsList: React.FC<Props> = ({ goods, errors }) => {
+  if (goods.length > 0) {
+    return (
+      <ul>
+        {goods.map(good => (
+          <li
+            key={good.id}
+            style={{ color: good.color }}
+          >
+            {good.name}
+          </li>
+        ))}
+      </ul>
+    );
+  }
+
+  if (!errors && goods.length === 0) {
+    return (
+      <div className="no-list">
+        No goods in list, click load button to see the list
+      </div>
+    );
```

Comment:
pass to GoodsList just goods, use conditional rendering with ternary to render goods if they are present or error